src/legacy/core_plugins/data/public/timefilter/timefilter.ts

```diff
@@ -195,10 +203,9 @@ export class Timefilter {
   };
 
   /**
-   * Added to allow search_poll to trigger an auto refresh event.
-   * Before this change, search_poll used to access a now private member of this instance.
+   * Trigger an auto refresh event
    */
-  public notifyShouldFetch = () => {
+  private notifyShouldFetch: TimerHandler = () => {
```

Comment:
Can you please remove this method and just trigger the observable from `setRefreshInterval`?

Reply:
I could remove it, yes, but I'd still need to declare it as a `TimerHandler` otherwise Typescript complains.

Reply:
Turns out if I added `window.setInterval` then Typescript appropriately associates the types.